### PR TITLE
make exitDuration governable

### PR DESCRIPTION
**Pull request by johannbarbie**
Resolves https://github.com/leapdao/leap-contracts/issues/93

### CODE REVIEW

File: contracts/ExitHandler.sol [hunk 1]
```diff
@@ -173,10 +173,22 @@ contract ExitHandler is DepositHandler {
     exitStake = _exitStake;
   }
 
+  function getExitStake() public view returns (uint256) {
```
Comment (troggy) [inline contracts/ExitHandler.sol]: why getters? Can't we just call `.exitStake()` since it is public?

Comment (eezcjkr) [inline contracts/ExitHandler.sol]: This changes the contract interface. Before I would call contract.exitStake(), now I have to call contract.getExitStake(). Why does this need to be done? And if we have to do it, we have to do the necessary changes in the node and bridgeUI also.

File: contracts/ExitHandler.sol [hunk 2]
```diff
@@ -38,8 +38,8 @@ contract ExitHandler is DepositHandler {
     uint256 stake;
   }
 
-  uint256 public exitDuration;
-  uint256 public exitStake;
+  uint256 exitDuration;
+  uint256 exitStake;
```
Comment (eezcjkr) [inline contracts/ExitHandler.sol]: See below.

**Comment by johannbarbie** [comment]
love the spirit. moved back to public modifier

**Pull request merged**